### ISSUE TITLE
help(WCAG20_Html_HasLang) add a "to-do" bullet to the help that `lang` also applies to `iframe`

### ISSUE THREAD

**Issue by philljenkins** [IBMa/equal-access#1424]
This rule correctly detects when an iFrame doesn't have the language setting
     [WCAG20_Html_HasLang.ts](https://github.com/IBMa/equal-access/blob/master/accessibility-checker-engine/src/v4/rules/WCAG20_Html_HasLang.ts) (or renamed html_lang_exists)

However, we got user feedback that they didn't know that their iFrames also need the language set. They assumed incorrectly that iFrames inherit the lang from the parent document.

To make this more explicit, let's add a to-do item to the Help for this case
Link to current Help file at [WCAG20_Html_HasLang.html](https://github.com/IBMa/equal-access/blob/master/accessibility-checker-engine/help-v4/en-US/WCAG20_Html_HasLang.html)
> - Page detected as HTML, but does not have a 'lang' attribute

Proposed bullet to add to Help list of "**_What to do:_**"
- **Or**, if an `iframe `is detected, add the `lang` attribute to the `iframe` element because documents loaded into iframes are still separate documents and do not inherit language settings from the parent document containing the iframe element.

See [Slack thread for background discussion](https://ibm-studios.slack.com/archives/GSGN38DQE/p1682354112586639)

**Comment by shunguoy** [comment]
triage: Phill will contact the user to see if we can get the original page to figure out the cause.

**Comment by philljenkins** [comment]
Sent Slack msg to Trish Smith to get a copy of page for testing.
https://ibm-studios.slack.com/archives/C036P1CTN/p1682265513148839

**Comment by shunguoy** [comment]
Triage: the user page has been updated since the issue was opened and the issue was fixed with accordion display:none that fixes the error. 
  The issue is closed.

**Comment by philljenkins** [comment]
Phill update the slack thread as well.
